src/support/azext_support/azext_metadata.json

```diff
@@ -1,4 +1,5 @@
 {
-    "azext.isPreview": true,
-    "azext.minCliCoreVersion": "2.0.81"
+    "azext.isPreview": false,
+    "azext.minCliCoreVersion": "2.0.81",
+    "azext.maxCliCoreVersion": "2.1.0"
```

Comment:
current CLI Core version is 2.2.0, naming 2.1.0 will disable the extension from that version of CLI.
Please remove it. or change it to a higher version like 3.0.0